pkg/skaffold/constants/constants.go

```diff
@@ -39,7 +39,7 @@ const (
 
 	DefaultKustomizationPath = "."
 
-	DefaultBusyboxImage = "busybox"
+	DefaultBusyboxImage = "gcr.io/k8s-skaffold/busybox"
```

Comment:
shd we follow the same naming convention as `skaffold-debug-support` and change this to `gcr.io/k8s-skaffold/skaffold-helpers/busybox`

Reply:
isn't it `DefaultDebugHelpersRegistry = "gcr.io/k8s-skaffold/skaffold-debug-support"` also?